mmv1/third_party/terraform/services/networkmanagement/resource_network_management_vpc_flow_logs_config_test.go

```diff
@@ -182,6 +182,176 @@ resource "google_network_management_vpc_flow_logs_config" "example" {
 	return fmt.Sprintf("%s\n\n%s\n\n", vpcFlowLogsCfg, testAccNetworkManagementVpcFlowLogsConfig_baseResources(context))
 }
 
+func TestAccNetworkManagementVpcFlowLogsConfig_network(t *testing.T) {
```

Comment:
We'll need to make these tests as beta-only, given `network` and `subnet` are beta-only fields. 
Details at https://googlecloudplatform.github.io/magic-modules/test/test/#add-an-update-test step 5:

If beta-only fields are being tested, do the following:

- Change the file suffix to `.go.tmpl`
- Wrap each beta-only test in a separate version guard: `{{- if ne $.TargetVersionName "ga" -}}...{{- else }}...{{- end }}`
- In each beta-only test, ensure that the TestCase sets `ProtoV5ProviderFactories: acctest.ProtoV5ProviderBetaFactories(t)`
- In each beta-only test, ensure that all Terraform resources in all configs have `provider = google-beta` set